packages/jira-adapter/src/change_validators/workflow_properties.ts

```diff
@@ -0,0 +1,56 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+import _ from 'lodash'
+import { ChangeValidator, getChangeData, isInstanceChange, SeverityLevel } from '@salto-io/adapter-api'
+import { collections } from '@salto-io/lowerdash'
+import { isWorkflowInstance, Status, Transition } from '../filters/workflow/types'
+import { Values } from '../../../adapter-api/src/values'
+import { WORKFLOW_TYPE_NAME } from '../constants'
+
+
+const { awu } = collections.asynciterable
+
+const getPropertiesKeys = (statusesOrTransitions: Status[]| Transition[]):
+  Array<string>[] => Array.from(statusesOrTransitions).map((param: Transition | Status) =>
```

Comment:
this is a bit confusing, I think either
```suggestion
  string[][] => Array.from(statusesOrTransitions).map((param: Transition | Status) =>
```
or 
```suggestion
  Array<Array<string>> => Array.from(statusesOrTransitions).map((param: Transition | Status) =>
```
will work

Reply:
agree, changed to ```string[][]```